### ISSUE TITLE
Failed installing version 2.1.3: messages/core: getDirectoryContents: does not exist

### ISSUE THREAD

**Issue by codygman** [Happstack/happstack-authenticate#1]
``` shell
Happstack/Authenticate/Core.hs:1:1:
    Exception when trying to run compile-time code:
      messages/core: getDirectoryContents: does not exist (No such file or directory)
    Code: mkMessageFor
            "HappstackAuthenticateI18N" "CoreError" "messages/core" ("en")
cabal: Error: some packages failed to install:
happstack-authenticate-2.1.3 failed during the building phase. The exception
```

This also affected being able to install clckwrks, though I may be able to work around it and use an older version. Also, I created a fresh sandbox and attempted to install happstack-authenticate.

**Comment by stepcut** [comment]
This should be fixed in happstack-authenticate-2.1.4. Although happstack-authenticate is built by a continuous integration system, it builds by checking out the source code, rather than building from the `cabal sdist` generated tarball. So it did not catch the fact that data-files where missing from the .cabal file.

In other news, I just now realized how to fix that flaw in the CI server -- something that has been bothering me for months.

Also, I am now reminded that happstack-authenticate-2.x does not automatically migrate accounts from happstack-authenticate-0. You need to run an external migration tool. I should document that...

**Comment by codygman** [comment]
Looks like it's fixed, I can install the clckwrks-example-dot-org code.

EDIT: I'm very glad I could help you unearth those bugs!
